openquake/hazardlib/tests/geo/surface/simple_fault_test.py

```diff
@@ -340,6 +340,26 @@ def test_hypocentre_index_2ndpatch(self):
         # The value used for this test is by hand calculation
         self.assertEqual(index, 2)
 
+    def test_hypocentre_index_large_mesh_spacing(self):
```

Comment:
This test is only indirectly covering the method you're modifying the most. It would be desirable to have also a test directly covering the get_fault_patch_vertices method.